spotinst/resource_spotinst_credential_aws.go

```diff
@@ -0,0 +1,105 @@
+package spotinst
```

Comment:
change the file name

Reply:
Updated file name to resource_spotinst_credentials_aws.go